docs/pages/web/modalalert.js

```diff
@@ -79,64 +70,143 @@ export default function ModalAlertPage({ generatedDocGen }: {| generatedDocGen:
         </MainSection.Subsection>
       </MainSection>
       <MainSection name="Best practices">
-        <MainSection.Subsection columns={2}>
+        <MainSection.Subsection>
           <MainSection.Card
-            cardSize="md"
+            cardSize="lg"
             type="do"
             description="Clearly communicate what response is expected and make the action simple and straightforward, such as clicking/tapping a button to confirm."
+            sandpackExample={
+              <SandpackExample
+                code={doClearCommunicate}
+                name="Clear communicate example"
+                hideEditor
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
           />
           <MainSection.Card
-            cardSize="md"
-            type="don't"
-            description="Use language that makes it hard to understand what action is being taken, while adding additional actions that may take the user out of their existing context."
+            cardSize="lg"
+            type="do"
+            description="Use to overlay Page content. ModalAlerts should be horizontally and vertically centered on the screen."
```

Comment:
Is this one required? I dont think this is specific for ModalAlert but for all Modals?

Reply:
Good call, removed